### DISCUSSION TITLE
station display problem

### DISCUSSION THREAD

**Discussion by bamyers99** [transitland/transitland#380] [Q&A]
In the current Firefox and Chrome browsers, the following stations briefly display the part above the Summary, Departures, Sources tabs, before the page is blanked.
https://www.transit.land/stops/s-dp3wjxychb-chicagounionstation
https://www.transit.land/stops/s-dr5ru4qgu5-nymoynihantrainhallatpennstation
https://www.transit.land/stops/s-dqcjr8vxkm-washingtonunionstation

User agents:
Mozilla/5.0 (X11; Ubuntu; Linux x86_64; rv:105.0) Gecko/20100101 Firefox/105.0
Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/106.0.0.0 Safari/537.36

**Comment by drewda** [comment]
Thanks for reporting this bug. We've reproduced the problem and will look into fixes for this.
